components/display_drivers/include/gc9a01.hpp

```diff
@@ -332,62 +327,28 @@ class Gc9a01 {
     set_drawing_area(x, y, x + width, y + height);
 
     // Write the color data to controller RAM
-    send_command(Command::ramwr);
     uint32_t size = width * height;
     static constexpr int max_bytes_to_send = 1024 * 2;
     uint16_t color_data[max_bytes_to_send];
     memset(color_data, color, max_bytes_to_send * sizeof(uint16_t));
     for (int i = 0; i < size; i += max_bytes_to_send) {
-      int num_bytes = std::min((int)(size - i), (int)(max_bytes_to_send));
-      send_data((uint8_t *)color_data, num_bytes * 2);
+      int num_bytes = std::min(static_cast<int>(size - i), (int)(max_bytes_to_send));
+      write_command_(static_cast<uint8_t>(Command::ramwr), reinterpret_cast<uint8_t *>(color_data),
```

Comment:
It's a little bit less efficient than the previous code (since previously we'd only send the command once), but I suppose that's OK for this since the bulk of the time will still be spend sending the color data.

Perhaps it would make sense to make the `command` parameter a `std::optional`? so that the `write_command_` function that's provided can handle the case where we don't want to send a new command but instead are just continuing to send more data?

Reply:
Similar to the above, since the code works and is cleaner, I'll resolve this PR and if we want later we can make this a future PR.